docs/pelion_ready/device-management-config.md

```diff
@@ -84,13 +86,19 @@ To run the tests:
 $ mbed test -t <TOOLCHAIN> -m <BOARD> -n simple*dev*connect --run -v
 ```
 
-## 2. Bootloader configuration
+## Bootloader configuration
 
-After you've successfully passed the "Connect" tests as described above, you can enable firmware update feature by adding a bootloader to your application.
+When the "connect" tests have completed successfully, you can enable the firmware update feature by adding a bootloader to your application.
```

Comment:
Completing and passing tests are not the same thing

Reply:
Right again. Needs to be passed.